backend/src/Designer/Program.cs

```diff
@@ -330,6 +331,7 @@ void Configure(IConfiguration configuration)
 
     app.MapHealthChecks("/health");
     app.MapHub<PreviewHub>("/previewHub");
+    app.MapHub<SyncHub>("/sync-hub");
```

Comment:
In general the guidlines are to use hyphens to improve readability or URI.
I didn't wanted to change the other endpoint url since it was out of scope.
But I guess best whould be to grop them under /hubs/sync and /hubs/preview. We can create a task for that change.